### ISSUE TITLE
Interchange { } and {!  !}

### ISSUE THREAD

**Issue by AndreVanDelft** [AndreVanDelft/scala#70]
{ } looks more tiny than {!  !}, but up to now the former marks a "fat" atomic action, whereas the latter a so-called "tiny" code fragment. This should be the other way round; then all atomic actions would have at least one other symbol next to the braces, as in {? ?}, {\* *}, {. .}, {... ...}.

**Comment by anatoliykmetyuk** [comment]
I don't think it is worthwhile doing. A purely cosmetic change will break the entire codebase we have by now. Also, a normal code fragment is the one that is used most frequently - hence, it must be the smallest one to type.

**Comment by AndreVanDelft** [comment]
FTTB we can leave it as it is.
I think though that it would be better if the most frequently used code would be the tiny code fragment, rather than the atomic action. The latter requires much more processing power, and in many cases it may well be replaced by a tiny code fragment, AFAIK.
I'd have to read through all existing examples to see whether this is really true.
